### PR TITLE
CU-864efmcy5 - Changes on neo3-invoker and neon-invoker to fix argume…

### PR DESCRIPTION
**Pull request by raulduartep**
…nts format to follow the standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,26 +10,33 @@
...
 ```bash
 npm i @cityofzion/neo3-invoker
 ```
 
 ### Documentation
+
 Checkout the auto-generated [Docs](https://htmlpreview.github.io/?https://raw.githubusercontent.com/CityOfZion/neo3-invoker/master/packages/neo3-invoker/docs/modules.html)
 based on the [Code](packages/neo3-invoker/src/index.ts).
 
+> If you are unsure about the expected value for each argument type, [Checkout these examples](packages/neo3-invoker/ARGUMENTS_EXAMPLES.md)
```
Comment (melanke) [inline README.md:30]: Is this link correct?

File: packages/neo3-invoker/README.MD [hunk 2]
```diff
@@ -10,9 +10,211 @@
...
 Neo3-Invoker is a specification of how SmartContract client SDKs can interact with different invoker libraries such Neon-JS or WalletConnect.
 
 With Neo3-Invoker a SDK don't need to reimplement the wheel and can focus on the SmartContract communication format.
 Taking advantage of the declarative nature of the specification, the SDK can be used with different invoker libraries, it only depends on what the dApp developer wants to use.
 
 Visit the [main page](../../README.md) of the project.
+
+## Arguments examples:
+
+For these examples, we are using the [NeonInvoker](../neon-invoker/README.md) that is an implementation of Neo3Invoker, so we are starting from the point that NeonInvoker is already initialized.
```
Comment (melanke) [inline packages/neo3-invoker/README.MD:23]: Actually, we are not using neon-invoker. It seems these examples below could be for any neo3-invoker. Right?

File: packages/neo3-invoker/README.MD [hunk 3]
```diff
@@ -10,9 +10,211 @@
...
+
+### Any
+
+It is expecting receive a HEX string as value.
```
Comment (melanke) [inline packages/neo3-invoker/README.MD:27]: It is expecting `to` receive a HEX string as value `on args`.

File: packages/neo3-invoker/README.MD [hunk 4]
```diff
@@ -10,9 +10,211 @@
...
+
+```ts
+const hexValue = 'HEX string'
```
Comment (melanke) [inline packages/neo3-invoker/README.MD:30]: this example doesn't help at all, we need a hex string example

File: packages/neo3-invoker/README.MD [hunk 5]
```diff
@@ -10,9 +10,211 @@
...
+invoker.testInvoke({
+  invocations: [
+    {
+      operation: 'method',
+      scriptHash: '0x0000000000000000000000000000000000000000',
```
Comment (melanke) [inline packages/neo3-invoker/README.MD:35]: this long scriptHash is a little bit distracting but I don't know if there is a way to improve this, let me know if you have any idea.

File: packages/neo3-invoker/README.MD (hunk 3 above)
Comment (melanke) [reply packages/neo3-invoker/README.MD:27]: Apply this change to the other parts as well

File: packages/neo3-invoker/README.MD [hunk 6]
```diff
@@ -10,9 +10,211 @@
...
+      args: [{ type: 'Boolean', value: booleanValue }],
+    },
+  ],
+})
+```
+
+### PublicKey
+
+It is expecting receive a HEX string as value. Compressed or Uncompressed format are allowed.
```
Comment (melanke) [inline packages/neo3-invoker/README.MD:78]: Compressed or Uncompressed format`s` are allowed

File: README.md (hunk 1 above)
Comment (raulduartep) [reply README.md:30]: It's wrong, I will fix.

File: packages/neo3-invoker/README.MD (hunk 5 above)
Comment (raulduartep) [reply packages/neo3-invoker/README.MD:35]: I was using 0x0. Lucas recommended me to use this way for better understanding

File: packages/neo3-invoker/README.MD (hunk 2 above)
Comment (raulduartep) [reply packages/neo3-invoker/README.MD:23]: It is true. I will change to: "For these examples, we are starting from the point where an invoker has already been initialized.". What do you think ?

File: packages/neon-invoker/src/index.ts [hunk 7]
```diff
@@ -173,30 +176,33 @@ export class NeonInvoker implements Neo3Invoker {
...
+          return sc.ContractParam.string(a.value)
         case 'Boolean':
-          return sc.ContractParam.boolean(a.value ?? false)
+          return sc.ContractParam.boolean(a.value)
         case 'PublicKey':
-          return sc.ContractParam.publicKey(a.value ?? '')
+          return sc.ContractParam.publicKey(a.value)
         case 'Address':
+        case 'ScriptHash':
+          return sc.ContractParam.hash160(Neon.u.HexString.fromHex(a.value))
```
Comment (melanke) [inline packages/neon-invoker/src/index.ts:195]: vc mudou o comportamento do Address mas só pra ser mais abrangente né? Funciona do jeito antigo, correto?

File: packages/neo3-invoker/README.MD (hunk 2 above)
Comment (melanke) [reply packages/neo3-invoker/README.MD:23]: Good

File: packages/neo3-invoker/README.MD (hunk 5 above)
Comment (melanke) [reply packages/neo3-invoker/README.MD:35]: maybe 
```json
invocations: [
    {
      operation: 'method',
      scriptHash: contractSH,
```

File: packages/neon-invoker/src/index.ts (hunk 7 above)
Comment (raulduartep) [reply packages/neon-invoker/src/index.ts:195]: It was my error, I forgot that when you don't pass a case block, it runs the next block.

File: packages/neo3-invoker/README.MD (hunk 5 above)
Comment (melanke) [reply packages/neo3-invoker/README.MD]: I've commited a change related to this topic: https://github.com/CityOfZion/neo3-invoker/pull/6/commits/8934df8ce268156f8ce153bb1254f28e943971f1

**Pull request merged**